Schemata/sarif-schema.json

```diff
@@ -783,7 +750,7 @@
 
         "workingDirectory": {
           "description": "The working directory for the analysis rool run.",
```

Comment:
>rool [](start = 65, length = 4)

Hah, while we're at it, `s/rool/tool/g`.

Reply:
nice catch. ran content through spell and grammar check and found a couple other double spaces and term simplifications.

---
In reply to: [215672927](https://github.com/oasis-tcs/sarif-spec/pull/236#discussion_r215672927) [](ancestors = 215672927)